orttraining/orttraining/training_api/include/interfaces.h

```diff
@@ -1,24 +1,190 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+/**
+ * @brief Temporary classes bridging between public C interfaces and internal ones.
```

Comment:
so will this be replaced by the C_API PR?

Reply:
Yes, it will be replaced by Ashiwini's implementation.

Reply:
As talked with @askhade, this will be deprecated very soon,.